package.json

```diff
@@ -48,12 +48,13 @@
     "@chainlink/contracts": "^0.3.1",
     "@ethersproject/providers": "^5.5.3",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
-    "@openzeppelin/contracts": "^4.4.2",
+    "@openzeppelin/contracts": "4.6.0",
     "@openzeppelin/test-environment": "^0.1.7",
     "@openzeppelin/test-helpers": "^0.5.15",
     "@uniswap/lib": "^1.1.2",
     "@uniswap/v2-core": "^1.0.1",
     "@uniswap/v2-periphery": "^1.1.0-beta.0",
+    "async": "2.6.4",
```

Comment:
Where's this being used? Doesn't look like it comes with typings so it won't work well natively with Typescript.

Reply:
dependabot was saying we have a high security issue because async was less than version 2.6.4, so I upgraded to fix this.